pubspec.lock

```diff
@@ -173,10 +173,10 @@ packages:
     dependency: transitive
     description:
       name: collection
-      sha256: f092b211a4319e98e5ff58223576de6c2803db36221657b46c82574721240687
+      sha256: "4a07be6cb69c84d677a6c3096fcf960cc3285a8330b4603e0d463d15d9bd934c"
```

Comment:
@NanoNish upgrade your flutter to latest version. don't downgrade the dependencies.

Reply:
yessir